control-plane/rest/tests/v0_test.rs

```diff
@@ -366,66 +366,72 @@ async fn client_test(mayastor1: &NodeId, mayastor2: &NodeId, test: &ComposeTest,
     let volume = client
         .volumes_api()
         .put_volume_target(
-            &volume.uuid.to_string(),
+            &volume.state.unwrap().uuid.to_string(),
             mayastor1.as_str(),
             models::VolumeShareProtocol::Nvmf,
         )
         .await
         .unwrap();
-    let nexus = volume.children.first().unwrap();
+    let volume_state = volume.state.expect("No volume state");
```

Comment:
this reads a bit funny, "expect no volume state" :)

Reply:
Changed message to make it more clear.